website/docs/examples/motor_state/README.md

```diff
@@ -21,33 +19,77 @@ The [**Amiga Motor State Stream**](https://github.com/farm-ng/farm-ng-amiga/blob
 example is a basic way of showing how to access and decode the
 `MotorState` values streamed by the canbus service.
 
+You can either run this example directly on a brain by `ssh`'ing in, or use your local PC.
+If using your local PC, it should be either connected to the same local network as the brain
+or linked to it through tailscale.
+
+The requirements to run this example are to have a
+[**farm-ng brain**](/docs/brain/) running the `canbus service`, without the physical E-STOP pressed.
+
 :::info
 There will be no `/motor_states` stream if your amiga is e-stopped by a physical e-stop press.
 The e-stop cuts the power to the motors, so they do not send their state on the CAN bus.
 :::
 
-## Setup
+## 1. Install the [farm-ng Brain ADK package](/docs/brain/brain-install)
+
+## 2. Install the example's dependencies
+
+### Setup
+
+```bash
+cd farm-ng-amiga/
+```
+
+:::tip Recommended
 
-Create first a virtual environment
+Create a virtual environment
 
 ```bash
-cd farm-ng-amiga
 python3 -m venv venv
 source venv/bin/activate
 ```
 
-## Install
+:::
+
+### Install
 
 ```bash
 cd py/examples/motor_states_stream
 pip install -r requirements.txt
 ```
 
-## Run example
+## 3. [Optional] Modify the Service Config
+
+:::info
+You can skip this if running the example `ssh`'d into your brain.
+:::
+
+To connect with a `gRPC` client over Wifi,
+you must change the `host` field in `service_config.json` from localhost to your robot's name
+(e.g., `element-vegetable`).
```

Comment:
I think that this only works for us (developers part of the engineers group on tailscale). If I'm not mistaken, customers would need to use the full tailscale name e.g., `element-vegetable.tail0be07.ts.net`

Reply:
Even if they are on the same wifi? This is something we have documented throughout...

Reply:
I don't have a way of testing, since I do not have a robot I can be on the same network as. Can we tap @OloFuchs for help here?

Reply:
> Even if they are on the same wifi? This is something we have documented throughout...

That's a good point. Maybe you can omit the full tailscale name if you're on the same network, but not if on different networks.

@OloFuchs to test for both conditions?